.github/workflows/kanidm_book.yml

```diff
@@ -67,3 +70,11 @@ jobs:
           ln -s ${{ needs.generate_tags.outputs.latest}}/ stable && \
           cd ..
           ls -la docs/
+
+      - name: Deploy
```

Comment:
What does this do?

Reply:
actually pushes it to github page, somehow that didn't get re-added